### PR TITLE
Update laptop script for our use

### PR DESCRIPTION
**Pull request by srt32**
This PR adapts https://github.com/18F/laptop to better fit our use.

There's a quite a bit of fodder here. @adamhadani, do you think we can start with roughly what we have here to get going and adjust as we go? Should we s/python3/python2`? Other major issues?

### CODE REVIEW

File: circle.yml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+dependencies:
```
Comment (srt32) [inline circle.yml]: @adamhadani once we get Circle set up, we should have bash checking running in CI, which will be nice.

**Comment by adamhadani** [comment]
@srt32 - nice re running this in a CI ! re python - should be 2 yes, otherwise looks like good start to me, i will go in depth more towards weekend

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
+## Welcome!
+
+We're so glad you're thinking about contributing toopen source project!  If
```
Comment (adamhadani) [inline CONTRIBUTING.md]: some typo here, + should we change the text to reflect joining 'globality' ?

Comment (srt32) [inline CONTRIBUTING.md]: Nice, updated in bb5caa8

**Comment by srt32** [comment]
s/python/python3 in 51940c3

**Comment by srt32** [comment]
@adamhadani not sure if you saw, but I don't have permissions for the Circle setup. Could you add this project?

File: mac [hunk 3]
```diff
@@ -0,0 +1,169 @@
...
+brew services restart redis
+
+brew_install_or_upgrade 'imagemagick'
+brew_install_or_upgrade 'qt'
+
+brew_install_or_upgrade 'hub'
+# shellcheck disable=SC2016
+append_to_file "$HOME/.zshrc" 'eval "$(hub alias -s)"'
+
+brew_install_or_upgrade 'node'
```
Comment (adamhadani) [inline mac]: few questions for us to consider pre-Allison:
1. should we go ahead and install nvm for node.js runtime if it's the best practice these days?
2. various support tools for our common development tools (node.js / coffeescript / javascript / python) - should these be installed? stuff like npm packages (coffee-script, jshint, bower, grunt-cli/gulp), sass/compass (ruby gem - might need to install ruby support) , flake8 etc? or these are ok to your mind being installed on an as-needed basis based on particular project's README files (as is currently the case, e.g fermina's [README](https://github.com/globality-corp/fermina/blob/develop/README.md))

**Pull request merged**